org.eclipse.jdt.debug.ui/plugin.properties

```diff
@@ -335,5 +335,6 @@ OpenFromClipboardAction.label = Open from Clipboar&d
 OpenFromClipboardAction.tooltip = Opens a Java element or Java Stack Trace from Clipboard
 OpenFromClipboardAction.description = Opens a Java element or a Java stack trace from clipboard
 OpenFromClipboardAction.name = Open from Clipboard
-
 VariablesView.name = Variables
+
+DisableAllConditions.label = Disable All Conditions
```

Comment:
There are no "Conditions" to be disabled, there are Conditional Breakpoints. So the menu should be "Disable Conditional Breakpoints".